.github/workflows/breaking-change-alert.yaml

```
@@ -0,0 +1,156 @@
+name: Breaking Change Notifications
+
+on:
+  workflow_call:
+    inputs:
+      sender_login:
+        description: 'Sender login'
+        required: true
+        type: string
+      sender_avatar:
+        description: 'Sender avatar URL'
+        required: true
+        type: string
+      repo:
+        description: 'Repository'
+        required: true
+        type: string
+      pr_number:
+        description: 'Pull request number'
+        required: true
+        type: number
+      pr_title:
+        description: 'Pull request title'
+        required: true
+        type: string
+      pr_body:
+        description: 'Pull request body'
+        required: false
+        default: '_Empty PR description_'
+        type: string
+      pr_base_ref:
+        description: 'Base branch of the pull request'
+        required: true
+        type: string
+      pr_author:
+        description: 'Pull request author'
+        required: true
+        type: string
+      event_action:
+        description: 'Event action'
+        required: true
+        type: string
+      pr_merged:
+        description: 'Pull request merged status'
+        required: true
+        type: boolean
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  notify-breaking-changes:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Escape inputs for JSON
+        id: escape
+        env:
+          SENDER_LOGIN: ${{ inputs.sender_login }}
+          SENDER_AVATAR: ${{ inputs.sender_avatar }}
+          REPO: ${{ inputs.repo }}
+          PR_NUMBER: ${{ inputs.pr_number }}
+          PR_TITLE: ${{ inputs.pr_title }}
+          PR_BODY: ${{ inputs.pr_body }}
+          PR_BASE_REF: ${{ inputs.pr_base_ref }}
+          PR_AUTHOR: ${{ inputs.pr_author }}
+        run: |
+          escape_json() {
+            echo "$1" | jq -R -c . | awk '{print substr($0, 2, length($0) - 2)}'
```

I would additionally add an assertion that there are actually quotes there:

```suggestion
            echo "$1" | jq -R -c . | grep -E '^".*"$' | awk '{print substr($0, 2, length($0) - 2)}' && [ "${PIPESTATUS[2]}" -eq 0 ]
```

---

Fixed